specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/schema/kafka.schema.patch.json

```diff
@@ -78,6 +78,166 @@
                                         {
                                             "type": "string",
                                             "enum": [ "none", "json_patch" ]
+                                        },
+                                        "key":
+                                        {
+                                            "type": "object",
+                                            "additionalProperties": false,
+                                            "properties":
+                                            {
+                                                "type":
+                                                {
+                                                    "title": "Type",
+                                                    "type": "string",
+                                                    "enum":
+                                                    [
+                                                        "byte_buffer",
+                                                        "bytes",
+                                                        "double",
+                                                        "float",
+                                                        "integer",
+                                                        "avro",
+                                                        "json",
+                                                        "json_decoder",
+                                                        "list",
+                                                        "long",
+                                                        "short",
+                                                        "string",
+                                                        "uuid",
+                                                        "void"
```

Comment:
What does `void` validation mean?
(also in duplicated enum below)

Reply:
validate if value is null.